### ISSUE TITLE
The field "me" is not unique in type "RootQueryType", when creating a fresh project

### ISSUE THREAD

**Issue by Nikola-Milovic** [PotionApps/potionx#6]
Running `npx @potionapps/templates project` results in an error as follows

```
== Compilation error in file lib/test_notes_graphql/schema.ex ==
** (Absinthe.Schema.Error) Compilation failed:
---------------------------------------
## Locations
/mnt/hddstorage/files/programming/web/projects/test_notes/lib/test_notes_graphql/schemas/user/user_queries.ex:20

The field "me" is not unique in type "RootQueryType".

The field must have a unique name within that Object type; no two fields may share the same name.
---------------------------------------
## Locations
/mnt/hddstorage/files/programming/web/projects/test_notes/lib/test_notes_graphql/schema.ex:25

The field "me" is not unique in type "RootQueryType".

The field must have a unique name within that Object type; no two fields may share the same name.


    (absinthe 1.6.7) lib/absinthe/schema.ex:366: Absinthe.Schema.__after_compile__/2
    (stdlib 3.17) lists.erl:1267: :lists.foldl/3
    (elixir 1.13.0) lib/kernel/parallel_compiler.ex:346: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/7
```

Full logs 

```
npx @potionapps/templates project
✔ What directory name should your app have? (snake_case) · test_notes
✔ What email to you want to log in with? · at@gmail.com
✔ What is your local PostgreSQL username? · postgres
✔ What is your local PostgreSQL password? · password
✔ Install dependencies? (Y/n) · true
Installing Deps...
✔ Run database preparation? (Y/n) · true
Running database set up... (this may be long due to compilation)
Database set up failed. Log:
==> typed_struct
Compiling 2 files (.ex)
Generated typed_struct app
==> file_system
Compiling 7 files (.ex)
Generated file_system app
==> connection
Compiling 1 file (.ex)
Generated connection app
==> nimble_parsec
Compiling 4 files (.ex)
Generated nimble_parsec app
==> gettext
Compiling 1 file (.yrl)
Compiling 1 file (.erl)
Compiling 21 files (.ex)
Generated gettext app
===> Analyzing applications...
===> Compiling ranch
===> Analyzing applications...
===> Compiling telemetry
==> telemetry_metrics
Compiling 7 files (.ex)
Generated telemetry_metrics app
===> Analyzing applications...
===> Compiling telemetry_poller
==> decimal
Compiling 4 files (.ex)
Generated decimal app
==> jason
Compiling 10 files (.ex)
Generated jason app
==> castore
Compiling 1 file (.ex)
Generated castore app
==> mint
Compiling 1 file (.erl)
Compiling 23 files (.ex)
Generated mint app
==> redix
Compiling 11 files (.ex)
Generated redix app
==> assent
Compiling 31 files (.ex)
Generated assent app
==> db_connection
Compiling 14 files (.ex)
Generated db_connection app
==> ecto
Compiling 56 files (.ex)
Generated ecto app
==> dataloader
Compiling 4 files (.ex)
Generated dataloader app
==> absinthe
Compiling 1 file (.yrl)
Compiling 1 file (.erl)
Compiling 256 files (.ex)
Generated absinthe app
==> absinthe_relay
Compiling 19 files (.ex)
Generated absinthe_relay app
==> phoenix_pubsub
Compiling 11 files (.ex)
Generated phoenix_pubsub app
===> Analyzing applications...
===> Compiling cowlib
===> Analyzing applications...
===> Compiling cowboy
===> Analyzing applications...
===> Compiling cowboy_telemetry
==> mime
Compiling 1 file (.ex)
Generated mime app
==> postgrex
Compiling 64 files (.ex)
Generated postgrex app
==> ecto_sql
Compiling 26 files (.ex)
Generated ecto_sql app
==> plug_crypto
Compiling 5 files (.ex)
Generated plug_crypto app
==> plug
Compiling 1 file (.erl)
Compiling 41 files (.ex)
Generated plug app
==> absinthe_plug
Compiling 18 files (.ex)
Generated absinthe_plug app
==> phoenix_html
Compiling 8 files (.ex)
Generated phoenix_html app
==> ecto_network
Compiling 4 files (.ex)
Generated ecto_network app
==> plug_cowboy
Compiling 5 files (.ex)
Generated plug_cowboy app
==> phoenix
Compiling 66 files (.ex)
Generated phoenix app
==> absinthe_phoenix
Compiling 9 files (.ex)
Generated absinthe_phoenix app
==> phoenix_live_reload
Compiling 4 files (.ex)
Generated phoenix_live_reload app
==> phoenix_live_view
Compiling 22 files (.ex)
Generated phoenix_live_view app
==> phoenix_live_dashboard
Compiling 36 files (.ex)
Generated phoenix_live_dashboard app
==> potionx
Compiling 42 files (.ex)
Generated potionx app
==> phoenix_ecto
Compiling 7 files (.ex)
Generated phoenix_ecto app
==> test_notes
Compiling 34 files (.ex)

== Compilation error in file lib/test_notes_graphql/schema.ex ==
** (Absinthe.Schema.Error) Compilation failed:
---------------------------------------
## Locations
/mnt/hddstorage/files/programming/web/projects/test_notes/lib/test_notes_graphql/schemas/user/user_queries.ex:20

The field "me" is not unique in type "RootQueryType".

The field must have a unique name within that Object type; no two fields may share the same name.
---------------------------------------
## Locations
/mnt/hddstorage/files/programming/web/projects/test_notes/lib/test_notes_graphql/schema.ex:25

The field "me" is not unique in type "RootQueryType".

The field must have a unique name within that Object type; no two fields may share the same name.


    (absinthe 1.6.7) lib/absinthe/schema.ex:366: Absinthe.Schema.__after_compile__/2
    (stdlib 3.17) lists.erl:1267: :lists.foldl/3
    (elixir 1.13.0) lib/kernel/parallel_compiler.ex:346: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/7
```

**Comment by the-vinner** [comment]
Will sort this out shortly!

**Comment by the-vinner** [comment]
I've released 0.4.3 of the project generator (@potionapps/templates) which I believe resolves this issue (just ran it myself).

**Comment by Nikola-Milovic** [comment]
Yep, using `npx @potionapps/templates@latest project` worked for me
